### PR TITLE
Edit general information #1

### PR DESCRIPTION
**Pull request by MechanicKim**
In General Information, Design and style are changed.

Info.plist and project.pbxproj are changed because of Ionicons.ttf

package.json are changed.
I changed react-native-datepicker into react-native-modal-picker.

### CODE REVIEW

File: src/components/SignUp/EduForm.js [hunk 1]
```diff
@@ -108,62 +56,253 @@ class EduForm extends Component {
     this.props.onChangeText('concentration', 'name', this.props.id, text);
   }
 
-  onChangeStartYear(year) {
-    this.props.onChangeText('startYear', 'name', this.props.id, year);
-    this.setState({ startYear: year });
+  onChangeDegree() {
```
Comment (ace0625) [inline src/components/SignUp/EduForm.js:60]: comment out this with TODO if you're going to use later or not.

File: src/components/SignUp/EduForm.js [hunk 2]
```diff
@@ -108,62 +56,253 @@ class EduForm extends Component {
...
+
+  }
+
+  onChangeStartYear(data) {
+    const year = data.label;
+    const date = year + '-' + this.state.startDate.month;
+    this.props.onChangeText('start_date', '', this.props.id, year);
+    this.state.startDate.year = year;
+    this.setState({});
```
Comment (ace0625) [inline src/components/SignUp/EduForm.js]: wonder what the setState does.

File: src/components/SignUp/EduForm.js [hunk 3]
```diff
@@ -108,62 +56,253 @@ class EduForm extends Component {
...
+  }
+
+  onChangeStartMonth(data) {
+    const month = data.label;
+    const monthStr = month.length === 1 ? '0' + month : month;
+    const date = this.state.startDate.year + '-' + monthStr;
+    this.props.onChangeText('start_date', '', this.props.id, date);
+    this.state.startDate.month = monthStr;
+    this.setState({});
```
Comment (ace0625) [inline src/components/SignUp/EduForm.js]: you might wanna initialize all the data variables in the constructor in order to use setState.

File: src/components/SignUp/EduForm.js [hunk 4]
```diff
@@ -108,62 +56,253 @@ class EduForm extends Component {
...
+        visible
+        onRequestClose={toggleEdit}
+      >
+        <View style={styles.modalContainer}>
+          <View style={styles.formEditView}>
+            <View style={styles.formEditBottomLine}>
+              <TextInput
+                style={[styles.formName, styles.formEditName]}
+                defaultValue={this.state.name}
+                underlineColorAndroid="rgba(255, 255, 255, 0)"
```
Comment (ace0625) [inline src/components/SignUp/EduForm.js]: I think we've been using hex color.

File: src/components/SignUp/EduForm.js (hunk 1 above)
Comment (MechanicKim) [reply src/components/SignUp/EduForm.js:60]: I wrote 'TODO: ...'

File: src/components/SignUp/EduForm.js (hunk 3 above)
Comment (MechanicKim) [reply src/components/SignUp/EduForm.js]: Thank you for reviewing and opinion to resolve it.
I fixed it.

**Comment by ace0625** [comment]
Thanks Jason! LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/140/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**